include/boost/geometry/algorithms/detail/buffer/turn_in_piece_visitor.hpp

```diff
@@ -21,6 +21,7 @@
 #include <boost/geometry/algorithms/covered_by.hpp>
 #include <boost/geometry/algorithms/detail/disjoint/point_box.hpp>
 #include <boost/geometry/algorithms/detail/disjoint/box_box.hpp>
+#include <boost/geometry/algorithms/detail/dummy_geometries.hpp>
```

Comment:
it is unclear to me whether dummy geometries are used in this file

Reply:
Because they are used by algorithms and they shouldn't be used by the users, hence `detail`. If you think they should be moved somewhere. I'd be ok with it.

Reply:
Maybe it is cleaner to be included by the details of any algorithm using them, but I am also OK as is for now since this is a bit unrelated with this PR.